### ISSUE TITLE
I applied NFKD normalization on the Arabic dict and data. It solved the issue.

### ISSUE THREAD

**Issue by soufieneghribi** [PaddlePaddle/PaddleOCR#3845]
I applied NFKD normalization on the Arabic dict and data. It solved the issue.
I am using 50.000 synthetic image data. After 8000 iterations (93 epochs) the train accuracy is about 85% but the best evaluation score I got (over 4000 images) is 4%. 
The images I am using have 32px of height with a variable length. 
What could it possibly cause the low accuracy on validation?
![image](https://user-images.githubusercontent.com/26394060/131034612-117393c1-bfd7-4fc4-a07b-e0a8864b9401.png)

_Originally posted by @soufieneghribi in https://github.com/PaddlePaddle/PaddleOCR/issues/3675#issuecomment-906736269_

**Comment by tink2123** [comment]
It may be because your synthetic data is too simple and is too different from the validation set.
It is recommended to add some real data during training, or add more data augments.
